examples/bottle_example.py

```diff
@@ -58,6 +58,7 @@ def dateadd(value, addend, unit):
 
 
 # Return validation errors as JSON
+@error(400)
 @error(422)
 def error422(err):
```

Comment:
Should this be renamed (remove 422 in the name)?

Reply:
Good catch. Fixed.